### DISCUSSION TITLE
guide/intro/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#386] [docs-comment]
# guide/intro/

vuepress-theme-plume 是一个基于 VuePress 的主题，无论您是想写 生活类博客、技术类博客、 或者是 产品文档、知识库、系列文档 等，主题都可以满足您的需求。 主题针对 文本内容、图片内容 的表现做了大量的优化，特别是针对 Markdown 内容的语法 做了 丰富的扩展， 你可以很轻松的利用这些特性编写 美观、易读、表现力丰富...

https://theme-plume.vuejs.press/guide/intro/

**Comment by coding-er** [comment]
1221

**Comment by baicaizhale** [comment]
很好看

**Comment by Duangce** [comment]
真的大为震惊，太强大了，我不了解内部是怎么样的原理，对于我这个车辆工程专业的学生来说，简直太不可思议了。这是一个伟大的艺术品，既有丰富到目不暇接的各自功能，又有美观多彩、惊艳绝伦的设计。
